.zuul.yaml

```diff
@@ -57,12 +57,6 @@
           identity.v3_endpoint_type public
           service_available.swift false
           service_available.cinder false
-      cifmw_test_operator_tempest_workflow:
```

Comment:
I know we removed it just for the testing purposes:). But this cannot be removed as part of this PR.

Reply:
You are right, I have returned the test for this PR.

Reply:
Thanks!:)